packages/common/src/utils/mime.utils.ts

```diff
@@ -11,64 +11,127 @@ export function getMimeType(path: string, fallback = FALLBACK_MIME_TYPE) {
 }
 
 /**
- * Determines if a file is an image
+ * Determines if a file path points to an image
  * @param path A path to a given file on disk
  * @returns True if the file is an image
  */
-export function isImageFile(path: string) {
+export function isImagePath(path: string) {
   return getMimeType(path).includes("image");
 }
```

Comment:
Is the duplication here intentional?
My gut says that the following wouldn't be much of an overhead:

```suggestion
export function isImagePath(path: string) {
  return isImageMimeType(getMimeType(path));
}
```